### PR TITLE
new update with exponential distributions

### PR DESCRIPTION
**Pull request by Jules-Deschamps**
@alebrigant parfois je suis passé par des listes au lieu d'arrays pcq on considère un seul paramètre

### CODE REVIEW

File: geomstats/information_geometry/binomial.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+        Parameters
+        ----------
+        n_samples : int
+            Number of samples.
+            Optional, default: 1.
+
+        Returns
+        -------
+        samples : array-like, shape=[..., 1]
```
Comment (alebrigant) [inline geomstats/information_geometry/binomial.py:52]: shape=[..., ]

File: geomstats/information_geometry/binomial.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            Sample of points representing binomial distributions.
+        """
+        return gs.random.rand(n_samples)
+
+    def projection(self, point, atol=gs.atol):
+        """Project a point in ambient space to the open set.
```
Comment (alebrigant) [inline geomstats/information_geometry/binomial.py:58]: "Project a point in ambient space to the parameter space of binomial distributions" ?

File: geomstats/information_geometry/exponential.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        n_samples : int
+            Number of samples.
+            Optional, default: 1.
+        bound : float
+            Right-end ot the segment where exponential parameters are sampled.
+            Optional, default: 1.
+
+        Returns
+        -------
+        samples : array-like, shape=[...]
```
Comment (alebrigant) [inline geomstats/information_geometry/exponential.py:55]: shape=[..., n_samples]

**Review by alebrigant — COMMENTED** [review]
Great, congrats on your first PR ! Now you need to write test files, one for each file you have created, and then I will be able to properly review. For now, I have juste made a few comments.

File: geomstats/information_geometry/binomial.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+from scipy.stats import binom
+
+
+class BinomialDistributions(OpenSet):
+    """Class for the manifold of binomial distributions."""
+
+    def __init__(self, n_draws):
+        super(BinomialDistributions, self).__init__(dim=1, ambient_space=Euclidean(dim=1), metric = BinomialFisherRaoMetric(n_draws))
```
[the remark this answers is not in the document]
Comment (Jules-Deschamps) [reply geomstats/information_geometry/binomial.py]: Yes I am using black now thanks!

File: geomstats/information_geometry/binomial.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            Tolerance to evaluate positivity.
+            Optional, default: gs.atol
+
+        Returns
+        -------
+        belongs : array-like, shape=[...,]
+            Boolean indicating whether point represents a binomial
+            distribution.
+        """
+        belongs = len(point.shape) == 1
```
[the remark this answers is not in the document]
Comment (Jules-Deschamps) [reply geomstats/information_geometry/binomial.py:39]: I think it still works for a single point but I have changed the float to an array (if the user enters a float) now yes thanks, I listed some test cases in the test files

File: geomstats/information_geometry/exponential.py [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+from scipy.stats import expon
+
+
+class ExponentialDistributions(OpenSet):
+    """Class for the manifold of exponential distributions."""
+
+    def __init__(self):
```
Comment (alebrigant) [inline geomstats/information_geometry/exponential.py:18]: Add something like "This is the parameter space of exponential distributions, i.e. the half-line of positive reals."

File: geomstats/information_geometry/binomial.py [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+from scipy.stats import binom
+
+
+class BinomialDistributions(OpenSet):
+    """Class for the manifold of binomial distributions."""
+
```
Comment (alebrigant) [inline geomstats/information_geometry/binomial.py:17]: Add something like "This is the parameter space of binomial distributions with fixed number of draws, i.e. the unit segment."

**Review by alebrigant — COMMENTED** [review]
Some minor comments before reformatting the tests !

**Comment by ninamiolane** [comment]
Hello, nice work! 🎉 

It would be better to close this PR, and re-open another one, because we see many commits that are not yours on the current history and it can mess up our timeline (I am responsible for making this mistake many times before too...). Thanks!

**Pull request closed without merging**